xslt-ballerina/Package.md

```diff
@@ -1,5 +1,14 @@
 ## Package Overview
+The XSLT library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina
+</a> language.
 
 This package provides a function to transform the XML content to another XML/HTML/plain text using XSL transformations.
 
-For information on the operations, which you can perform with this module, see the below **Functions**. For examples on the usage of the operations, see the [XSLT Transformation Example](https://ballerina.io/learn/by-example/xslt-transformation.html).
\ No newline at end of file
+### Report Issues
+
+To report bugs, request new features, start new discussions, view project boards, etc. please visit Ballerina Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
+
+### Useful Links
+- Discuss the code changes of the Ballerina project in ballerina-dev@googlegroups.com.
```

Comment:
```suggestion
- Discuss code changes of the Ballerina project in ballerina-dev@googlegroups.com.
```

Reply:
Add link here.

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-xslt/pull/138/commits/97a6cdaeece38447d139b67db168d614b8278682